actions/conventional_commit/main.go

```diff
@@ -32,6 +32,14 @@ var allowedCommitTypes = map[string]struct{}{
 	"revert":   {},
 }
 
+// bypassAuthorEmails are the authors that are allowed to bypass the conventional commit
+// check.
+//
+// Note: Commits must have a valid GPG signature to bypass the check.
+var bypassAuthorEmails = map[string]struct{}{
+	"49699333+dependabot[bot]@users.noreply.github.com": {},
```

Comment:
Where is this email coming from? Is this the _only_ dependabot user possible? Link to docs if possible.

Reply:
There's not docs for this, but that number+username is the pattern for emails for Github Apps. So, I suspect that's it. That being said, I don't know of any docs that "prove" it.

Reply:
Yeah, I think a comment like:
```suggestion
	// Static email for dependabot app.
	"49699333+dependabot[bot]@users.noreply.github.com": {},
```
would suffice.